server/src/test/java/com/cloud/storage/VolumeApiServiceImplTest.java

```diff
@@ -1113,6 +1138,7 @@ public void isNotPossibleToResizeTestNoRootDiskSize() {
         prepareAndRunTestOfIsNotPossibleToResize(Type.ROOT, 0l, Storage.ImageFormat.QCOW2, false);
     }
 
+    // FR123 update this test case to check service offering strictness
     private void prepareAndRunTestOfIsNotPossibleToResize(Type volumeType, Long rootDisk, Storage.ImageFormat imageFormat, boolean expectedIsNotPossibleToResize) {
```

Comment:
Is this test case to be updated? remove the comment if not required / after updating it.